src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -131,8 +125,9 @@ public void parse_repeatedNonTagValue_failure() {
 
     @Test
     public void parse_optionalFieldsMissing_success() {
-        // zero tags
-        Person expectedPerson = new PersonBuilder(AMY).withTags().build();
+        // When optional fields (including category) are missing, default category should be used.
+        Person expectedPerson = new PersonBuilder(BOB)
+                .withCategory("Investor").build();
```

Comment:
```suggestion
        // When optional fields (including category) are missing, default category should be used.
        Person expectedPerson = new PersonBuilder(AMY)
                .withCategory("Other").build();
```
You're using Amy for the rest of the testcase